### DISCUSSION TITLE
哥啊,有直接http请求使用的api吗?

### DISCUSSION THREAD

**Discussion by Gloria-GK-406** [zhimin-dev/iptv-checker-rs#17] [Q&A]
请问这个项目有网络调用的API吗?直接使用http请求来使用服务

**Comment by GrandDuke1106** [comment]
有的，在[这里](/zhimin-dev/iptv-checker-rs/blob/main/src/web.rs#L60)

使用 get ``http://hostname/check/url-is-available?url=<m3u8url>&timeout=8000``

**Reply by Gloria-GK-406** [reply]
好的，谢谢啊

**Comment by zmisgod** [comment]
其实你只要获取最终的m3u文件就可以了，检测的任务交给后台处理，定时一小时或者一天检测一次。

如果你要检测某个m3u8的直播源能否使用，就用2楼回复的接口也可以。

**Reply by Gloria-GK-406** [reply]
好的，okok
